### PR TITLE
[AC-1122] Add new admin access to all collections and items settings 

### PR DESCRIPTION
**Pull request by shane-melton**
## Type of change

```
- [ ] Bug fix
- [X] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Add the checkbox for the new collection management setting created in this server PR https://github.com/bitwarden/server/pull/3379

## Code changes

- **apps/web/src/app/admin-console/organizations/settings/account.component:** Add the new checkbox and update the form group to support `allowAdminAccessToAllCollectionItems`
- **libs/common/src/admin-console/models/response/profile-organization.response.ts:** Add the new setting
- **libs/common/src/admin-console/models/response/organization.response.ts:** Add the new setting

## Screenshots

image

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

**Review by vincentsalucci — DISMISSED** [review]

**Review by LRNcardozoWDF — DISMISSED** [review]
LGTM!

File: apps/web/src/app/admin-console/organizations/settings/account.component.html [hunk 1]
```diff
@@ -59,6 +59,10 @@ <h1 bitTypography="h1" class="tw-mt-16 tw-pb-2.5">{{ "apiKey" | i18n }}</h1>
 >
   <h1 bitTypography="h1" class="tw-mt-16 tw-pb-2.5">{{ "collectionManagement" | i18n }}</h1>
   <p>{{ "collectionManagementDesc" | i18n }}</p>
+  <bit-form-control>
+    <bit-label>{{ "allowAdminAccessToAllCollectionItemsDesc" | i18n }}</bit-label>
+    <input type="checkbox" bitCheckbox formControlName="allowAdminAccessToAllCollectionItems" />
+  </bit-form-control>
```
Comment (eliykat) [inline apps/web/src/app/admin-console/organizations/settings/account.component.html:65]: v1 should be behind its own feature flag: `flexible-collections-v-1` (for some reason LaunchDarkly wasn't taking `v1`).

The timing might work out that MVP and v1 get released together anyway, but that's not a sure thing so we still want this flexibility.

**Review by eliykat — CHANGES_REQUESTED** [review]

**Review by eliykat — APPROVED** [review]

**Review by LRNcardozoWDF — APPROVED** [review]
Looks good to me!

**Review by vincentsalucci — DISMISSED** [review]
I also have a PR that touches the `account.component`. If yours get merged first, I'll handle the conflicts 🎉

**Review by vincentsalucci — APPROVED** [review]

**Pull request merged**